src/endpoints/journals/getJournals.ts

```diff
@@ -0,0 +1,20 @@
+import axios, { AxiosResponse } from 'axios';
+import { JournalsMessage } from './types';
+import { endpoints } from '../../constants';
+import { RequestConfig } from '../../types/RequestConfig';
+import { createParams } from '../../utils/createParams';
+
+// Return a list of journals in the Crossref database.
+export const getJournals = async (config?: Omit<RequestConfig, 'sort'>) => {
+  try {
+    const { BASE_URL, JOURNALS } = endpoints;
+    const params = createParams(config);
+    const url = BASE_URL + JOURNALS + params;
```

Comment:
In this file you're adding strings together with + operator, in other files you're adding strings together with interpolation and backticks ``